Client/multiplayer_sa/StdInc.h

```diff
@@ -11,28 +11,16 @@
 #include "SharedUtil.MemAccess.h"
 #include <stdio.h>
 
-#include <algorithm>
-#include <list>
-#include <map>
-#include <string>
-#include <vector>
-
```

Comment:
Keep standard C++ headers here if they're used in the project.

Reply:
they are not used